examples/SampleApp/src/components/ChatScreenHeader.tsx

```diff
@@ -104,7 +72,7 @@ export const ChatScreenHeader: React.FC<{ title?: string }> = ({
           />
         </RoundButton>
       )}
-      Title={isOnline ? undefined : NetworkDownIndicator}
+      Title={isOnline ? undefined : () => <NetworkDownIndicator titleSize={'large'} />}
```

Comment:
in general this is the correct way to do it. but otherwise from my other comments this seems unnecessary to have changed from what it was before
```suggestion
      Title={isOnline ? undefined : () => <NetworkDownIndicator titleSize='large' />}
```